README.md

```diff
@@ -92,7 +95,7 @@ docker run -d -p 9090:9090 --name prometheus -v $PWD/prometheus.yml:/etc/prometh
 
 #### Spans
 
-1. Uncomment `.AddOtlpExporter` below `builder.Services.AddOpenTelemetryTracing`, in the `TodoApi/OpenTelemetryExtensions.cs` file
+1. Configure Env var `OTEL_EXPORTER_OTLP_ENDPOINT` with the right endpoint URL to enable `.AddOtlpExporter` below `builder.Services.AddOpenTelemetryTracing`, in the `TodoApi/OpenTelemetryExtensions.cs` file
```

Comment:
`OTEL_EXPORTER_OTLP_ENDPOINT` looks like no so friendly env var for a .NET application using the config system. Is there any reason to use this specific env var or could it look like:  `OTLP__ENDPOINT`?

Reply:
> `OTEL_EXPORTER_OTLP_ENDPOINT` looks like no so friendly env var for a .NET application using the config system. Is there any reason to use this specific env var or could it look like: `OTLP__ENDPOINT`?

I decided to use the same variable that `OpenTelemetry.Exporter.OpenTelemetryProtocol` uses to configure the endpoint internally in the library. I can change/add one specific for the todoapi if you want to.

Reply:
Here's OpenTelemetry Protocol Exporter [doc](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#:~:text=4318%20%5B1%5D-,Env%20vars%3A%20OTEL_EXPORTER_OTLP_ENDPOINT,-OTEL_EXPORTER_OTLP_TRACES_ENDPOINT%20OTEL_EXPORTER_OTLP_METRICS_ENDPOINT%20OTEL_EXPORTER_OTLP_LOGS_ENDPOINT).

Reply:
Thanks!